wbooks/src/app/screens/BookDetail/components/CommentTable/component/CommentItem/styles.js

```diff
@@ -0,0 +1,36 @@
+import { StyleSheet } from 'react-native';
+import { darkGray, cyan } from '../../../../../../../constants/colors';
+
+export default StyleSheet.create({
+	container: {
+        alignItems: 'flex-start',
+        flexDirection: 'row',
+        backgroundColor: 'white',
+        borderRadius: 5,
+        padding: 15
+    },
+    userImage: {
+        width: 45,
+        height: 45,
```

Comment:
extract common size in a constant value in this file

Reply:
done